schema/deploy/tables/reporting_requirement.sql

```diff
@@ -0,0 +1,50 @@
+-- Deploy cif:tables/reporting_requirement to pg
+
+
+begin;
+create table cif.reporting_requirement
+(
+  id integer primary key generated always as identity,
+  due_date timestamptz,
```

Comment:
```suggestion
  report_due_date timestamptz,
  completion_date timestamptz,
  submitted_date timestamptz,
```

The completion date would be the date at which the tasks associated with the reporting requirement should be completed. For milestones, it's a different date than the report_due_date